client/src/components/App/Home/MakeBooking.tsx

```diff
@@ -78,19 +82,21 @@ const MakeBooking: React.FC<Props> = (props) => {
 
   // Global state
   const { state, dispatch } = useContext(AppContext);
-  const { user } = state;
+  const { config, user } = state;
 
   // Local state
   const [weeks, setWeeks] = useState<Week[]>([]);
   const [rows, setRows] = useState<Row[]>([]);
   const [selectedWeek, setSelectedWeek] = useState<number | undefined>();
   const [buttonsLoading, setButtonsLoading] = useState(true);
-  const [slideConfirm, setSlideConfirm] = useState(false);
-  const [todayWithParking, setTodayWithParking] = useState(false);
+  const [showTodayConfirmation, setShowTodayConfirmation] = useState(false);
+  const [showReasonConfirmation, setShowReasonConfirmation] = useState(false);
+  const [bookingDate, setBookingDate] = useState<Date | undefined>();
+  const [bookingParking, setBookingParking] = useState(false);
+  const [bookingReason, setBookingReason] = useState<string | undefined>();
```

Comment:
Just a thought ... this might be a good candidate for moving into its own component at some point to encapsulate the state